composer.json

```diff
@@ -21,6 +21,8 @@
         "yiisoft/yii-dataview": "^3.0@dev",
         "yiisoft/arrays": "^3.0@dev",
         "yiisoft/strings": "^3.0@dev",
```

Comment:
Not used at all:

- yiisoft/strings
- yiisoft/yii-dataview
- yiisoft/di
- yiisoft/event-dispatcher

Should be moved to require from require-dev:

- yiisoft/log
- yiisoft/view

Reply:
I will remove unused dependencies later